### DISCUSSION TITLE
[Feature] General Improvement-Ideas for the Mobile-App-Design

### DISCUSSION THREAD

**Discussion by dafeist-github** [immich-app/immich#7611] [Feature Request]
### The feature

Hi,

here are some design-changes I'd love to see in the mobile app:

-A setting/slider in the settings to make the thumbnails bigger
-A setting to hide the "cloud-sync" icon
-A setting for less distance between photos(the white in between)
-Make some photos/videos bigger, like in gphotos

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
You can already change the thumbnail size as well as hide the cloud sync icon from the mobile app setting under Photo Grid

**Reply by dafeist-github** [reply]
Oh yeah you are right there, just found it, but the distance between photos + make some thumbs bigger features do not exist yet
